docs/platform/howto/static-ip-addresses.rst

```diff
@@ -0,0 +1,108 @@
+Static IP addresses
+===================
+
+**Note: Each static IP addresses has an hourly price ranging from $0.025
+to $0.075 depending on region. The feature is recent and
+Aiven's**\ `pricing page <https://aiven.io/pricing>`__\ **doesn't
+include static IPs yet.**
+
+By default an Aiven service will use public IP addresses allocated from
+the cloud provider's shared pool of addresses for the cloud region. As a
+result, Aiven service nodes effectively get a random IP address. Because
+the addresses are shared with all other users of the cloud region, it's
+not feasible to create firewall rules to limit access to/from nodes
+using public IP addresses as the firewall would need to grant access to
+the cloud region's entire IP range granting access to all other users of
+that cloud region.
+
+You can however create IP addresses to your project. These addresses
+have a static IP for the lifetime of the address resource, and makes the
+addresses predictable for firewall rules. After creating the address it
+can be associated with an Aiven service. Once enough IP addresses have
+been associated with a service, you can turn on static IP use on the
+service.
+
+.. _h_ff06143f3e:
+
+Setting it up
+-------------
+
+| Start by creating IP addresses into the cloud region you intend to use
+  them for a service. The amount of IP addresses depends on the service
+  plan. You'll need 2x the number of IP addresses compared to the amount
+  of nodes in the plan. This is so maintenance upgrades and failovers
```

Comment:
Instead of "This is so maintenance upgrades and failovers can be handled without using non-static IP addresses"

maybe 

"We request a number of static IPs two times the amount of the cluster nodes, since it guarantees upgrades and failovers to be handled using only static IP addresses and not relying on random ones."